phpBB/viewtopic.php

```diff
@@ -142,9 +140,10 @@
 		}
 		else
 		{
+			$forum_id = $row['forum_id'];
 			$sql = 'SELECT topic_id, forum_id
 				FROM ' . TOPICS_TABLE . '
-				WHERE forum_id = ' . $row['forum_id'] . "
+				WHERE forum_id = ' . $forum_id . "
 					AND topic_moved_id = 0
 					AND topic_last_post_time $sql_condition {$row['topic_last_post_time']}
 					AND " . $phpbb_content_visibility->get_visibility_sql('topic', $row['forum_id']) . "
```

Comment:
As of `$forum_id = $row['forum_id'];` above, it can be used here as well:
```
AND " . $phpbb_content_visibility->get_visibility_sql('topic', $forum_id . "
```